### DISCUSSION TITLE
Clustercode full deployment steps and clustercode-admin

### DISCUSSION THREAD

**Discussion by rmansillasoto** [ccremer/clustercode#78] [Show and tell]
Helo Chris, 

First of all, thanks for sharing your work, I think clustercode it´s great and it deserves to give it a try as I see it's very interesting but Im a little bit newbie on k8s and don´t know how to make the full deployment for clustercode (I didn´t found it reading the docs), also I´m wondering how can I make clustercode and clustercode-admin working together.

It could be great if you can guide me a little bit in order to get the deployment ready for testing.

Thanks a lot for the info in advance :-)

Regards,
Raúl.

**Comment by ccremer** [comment]
Hi

thanks for your interest in my project and sorry for the late response.
First you have to know that clustercode is my private little project that I tinker with in my spare time. Clustercode v2 is a complete rewrite that is not compatible with clustercode v1 at all, including clustercode-admin. Furthermore, the installation instructions are, as you've found out, not existing. One with some Kustomize knowledge would be able to install it in a Kubernetes cluster though.

However, clustercode v2 is still just in a Proof-of-Concept quality. I made it work in my machine, but probably it doesn't work in your environment. Certain things like user IDs are hard coded at the moment and also API wise I cannot guarantee stability. So I doubt it's useful for you at this moment anyway even if you do manage to install it.

Moreover: Clustercode v2 is for me currently more of a technology playground than something I'm actually in need for. Since I also work as a software developer in my full-time job, I only rarely have interest in spending my free time in even more coding. So development is quite slow.

Long story short: It's unfortunately just too early to try clustercode v2.

You could try clustercode v1.3 with clustercode-admin, but it's dated and I cannot provide any further support for it I'm afraid. The docs would be here: https://ccremer.github.io/clustercode-docs/clustercode/1.3.3/index.html

Best regards

**Reply by rmansillasoto** [reply]
Thanks a lot for your feedback! Understood...just wanted to play with it and learn k8s meanwhile :-) I will take a look into v1.3 and see if I can get something.

Thanks again!
